client/lib/cart-values/test/index.js

```diff
@@ -23,9 +23,9 @@ describe( 'index', () => {
 		PREMIUM_PRODUCT,
 		THEME_PRODUCT;
 
-	beforeAll( () => {
-		cartValues = require( 'lib/cart-values' );
-		cartItems = cartValues.cartItems;
+	beforeAll( async () => {
+		cartValues = await import( 'lib/cart-values' );
+		cartItems = await import( 'lib/cart-values/cart-items' );
```

Comment:
I reviewed the file, and it doesn't look like we're mocking them or otherwise doing anything that would require a state reset. I moved this to static imports.